ibis/expr/datatypes/core.py

```diff
@@ -539,6 +487,8 @@ def __repr__(self):
         return f"{self.name}(cardinality={cardinality})"
 
     def to_integer_type(self):
```

Comment:
This method doesn't seem to be used anywhere, can we delete it?

Reply:
I defer it to a follow-up issue since I'm unable to run the impala tests locally.

Reply:
https://github.com/ibis-project/ibis/issues/4477